pkg/datasource/datasource_test.go

```diff
@@ -360,6 +378,15 @@ func TestDatasource(t *testing.T) {
 		require.Equal(t, "graf2 (AWS2)", response.Responses["A"].Frames[0].Fields[6].At(0))
 	})
 
+	t.Run("getInsightSummaries query with different region", func(t *testing.T) {
```

Comment:
What does "with different region" mean in this context? Different than the default region?

Reply:
yeah different from the default